abci/types/types.pb.go

```diff
@@ -3844,7 +3843,7 @@ func NewABCIClient(cc grpc1.ClientConn) ABCIClient {
 
 func (c *aBCIClient) Echo(ctx context.Context, in *RequestEcho, opts ...grpc.CallOption) (*ResponseEcho, error) {
 	out := new(ResponseEcho)
-	err := c.cc.Invoke(ctx, "/tendermint.abci.ABCI/Echo", in, out, opts...)
```

Comment:
This is the gRPC protocol breakage I'm talking about in https://github.com/cometbft/cometbft/issues/94#issuecomment-1456520852